addon/mixins/popover.js

```diff
@@ -42,9 +42,11 @@ export default Ember.Mixin.create(StyleBindingsMixin, BodyEventListener, {
   }).property('contentViewClass'),
   didInsertElement: function() {
     this._super();
-    this.snapToPosition();
-    this.set('visibility', 'visible');
-    return this.set('isShowing', true);
+    Ember.run.schedule('afterRender', this, () => {
```

Comment:
This fixed a double-render warning?

Reply:
Yes. Modifying properties in `didInsertElement`